src/qt/sendcoinsdialog.cpp

```diff
@@ -652,6 +652,16 @@ void SendCoinsDialog::updateFeeMinimizedLabel()
     }
 }
 
+CoinControlDialog* SendCoinsDialog::coinControlDialog()
+{
+    if (!m_coin_control_dialog) {
+        m_coin_control_dialog = new CoinControlDialog(platformStyle, this);
+        m_coin_control_dialog->setModel(model);
```

Comment:
Why not pass `model` to the `CoinControlDialog` constructor?

Reply:
I can make that change, but I think it's unrelated here.

Reply:
We use the `setModel()` approach in various places where the model isn't known at construction time, but that's not the case here. But agree it's not really related.